src/Execution/LighthouseRequest.php

```diff
@@ -4,7 +4,7 @@
 
 use Illuminate\Http\Request;
 
-class LighthouseRequest implements GraphQLRequest
+class LighthouseRequest extends BaseRequest
```

Comment:
@spawnia would we prefer this class to implement `GraphQLRequest` instead as that make it clearer to understand that this is actually an implementation of the interface.

Reply:
We don't do that with the directives and the `BaseDirectives`, so it would stay consistent and do the same here.